.github/workflows/test.yaml

```diff
@@ -38,10 +39,68 @@ jobs:
             reports/jest/junit.xml
             reports/junit/*.xml
           comment_mode: off
-      - name: Send Slack notification with job status.
-        if: always()
+  save-cache:
+    name: Save Cache
+    runs-on: ubuntu-20.04
+    permissions: {}
+    steps:
+      - name: Check out repository.
+        uses: actions/checkout@v3.0.2
+      - name: Save Docker images.
+        uses: ./
+        with:
+          key: docker-cache-test
+      - name: Build an empty test Docker image.
+        run: |
+          docker build --tag empty .
+  restore-cache:
+    name: Restore Cache
+    needs:
+      - save-cache
+    runs-on: ubuntu-20.04
+    permissions: {}
+    steps:
+      - name: Check out repository.
+        uses: actions/checkout@v3.0.2
+      - name: Load Docker images.
+        uses: ./
+        with:
+          key: docker-cache-test
+      - name: Verify Docker loaded empty image from cache.
+        run: |
+          description="$(docker inspect --format '{{ index .Config.Labels "description" }}' empty)"
+          [[ $description == "empty image" ]]
+  delete-cache:
+    name: Delete Cache
+    if: needs.save-cache.result == 'success'
+    needs:
+      - save-cache
+      - restore-cache
+    runs-on: ubuntu-20.04
+    permissions:
+      actions: write # for deleting cache
+    steps:
+      - name: Delete existing test cache.
+        run: >
+          curl
+          --request DELETE
+          --header "Accept: application/vnd.github.v3+json"
+          --header "Authorization: token $GITHUB_TOKEN"
+          "https://api.github.com/repos/ScribeMD/docker-cache/actions/caches?key=docker-cache-test&ref=$GITHUB_REF"
+  notify:
```

Comment:
Do you mean to make it part of the delete-cache job?  I recall that we had separated out the notify job since it was doing the notifications for all of the above jobs.  What am I missing??

Reply:
No, I mean literally just add a line that says `name: Notify` below this line.

Reply:
Gotcha.  That makes sense.  Thanks for clarifying